test/data/README.md

```diff
@@ -0,0 +1,43 @@
+# Data Test README
+
+## spreadsheets-health.test.ts
+
+The main test in this file ensures that registered spreadsheets are in sync with data that's checked in. This allows us to have spreadsheets that don't incur significant tech debt every time we need to convert them to JSON (or vice versa).
+
+The easiest mental model for what the test is doing is making sure the spreadsheet and JSON data are the same.
+
+More specifically every row in a registered spreadsheet must be in sync with checked-in JSON records in the repo. A row can:
+
+1. fail CollectedIncentive validation. This means it doesn't even meet our schema for collected data in spreadsheet form. These go in the `incentives_invalid_collected.json` file for that state.
+2. fail StateIncentive validation. These records are valid spreadsheet records but something is failing in the process of converting them to servable JSON. These go in the `incentives_invalid_state.json` file for that state.
+3. be valid, servable JSON. These go in the `incentives.json` file for that state.
+
+Eventually, we'd want all spreadsheet records go in category #3, but we're not there yet. In the meantime, categories #1 and #2 are for meeting the spreadsheets as they are and making sure they don't get worse (worse meaning more out of sync from our schemas and data).
+
+### Registering a state and fixing failures the first time
+
+To add a new state, flip the `runSpreadsheetHealthCheck` boolean in `scripts/incentive-spreadsheet-registry.ts` to true. Then run the test. It'll almost certainly fail.
```

Comment:
Not all states currently have this boolean set - should this read "set the `runSpreadsheetHealthCheck` boolean..." instead, to capture this distinction?

Reply:
Yes, set is more accurate terminology than fix. Will correct.